receiver/prometheusreceiver/metrics_receiver_test.go

```diff
@@ -28,6 +28,10 @@ import (
 
 // target1 has one gauge, two counts of a same family, one histogram and one summary. We are expecting the both
 // successful scrapes will produce all metrics using the first scrape's timestamp as start time.
+
+// With the 2nd page, we are simulating a reset (values smaller than previous), start_timestamp should be from
+// this run for the 2nd scrape, despite a failed scrape between 1st and 2nd page.
```

Comment:
Is this change necessary?  Isn't there a separate test that tests this scenario?  If not, what of the situation that was previously tested, i.e., a successful scrape after a failed scrape that is not a series reset?

Reply:
The scenario: _highervalue-failescrape-lowervalue_ was not tested before for summary and histogram. Receiver is not resetting the `start_timestamp` for all cumulative types as per this scenario, and this was not captured in these tests. 

Earlier this test case tested: _value-failedscrape-highervalue_ I have added this scenario in target3page3. Target3 checks the histogram and summary only. So I added this scenario there. 

For counter, both the scenarios- _highvalue-failedscrape-lowervalue_; _value-failedscrape-highervalue_ is tested in target2.